firmware/esp32_ttgo/main.cpp

```diff
@@ -0,0 +1,426 @@
+#include <Arduino.h>
+
+// required for LoRa and/or AsyncSDServer libraries?
+#include <FS.h>
+#include <SPI.h>
+
+#include <WiFi.h>
+#include <AsyncTCP.h>
+#include <ESPAsyncWebServer.h>
+#include <ESPmDNS.h>
+#include <AsyncSDServer.h>
+#include <SD.h>
+#include <SPIFFS.h>
+#include "SSD1306Wire.h"
+
+#include "config.h"
+
+// server
+#include "server/DisasterRadio.h"
+// client
+#include "client/OLEDClient.h"
+#include "client/HistoryRecord.h"
+#include "client/LoRaClient.h"
+#include "client/StreamClient.h"
+#include "client/TCPClient.h"
+#include "client/WebSocketClient.h"
+#include "client/GPSClient.h"
+// middleware
+#include "middleware/Console.h"
+#include "middleware/HistoryReplay.h"
+// history
+#include "history/HistorySD.h"
+#include "history/HistoryMemory.h"
+
+#include "DisasterHistory.h"
+#include "DisasterMiddleware.h"
+
+#define MDNS_NAME_PREFIX "disaster"
+#define HOST_NAME "disaster.radio"
+#define WIFI_AP_SSID_PREFIX "dstr "
```

Comment:
I would like to keep this as "disaster.radio" because it is free advertising, i.e. if some one see the ssid disaster.radio and searches for it, they will reach our website or this repo. This is approach is used by peoplesopen.net and libremesh (and I'm sure other community networks). I think you did this so it would fit on the OLED maybe? I would suggest just putting the MAC address on the OLED, dropping the SSID prefix.

Ultimately, the SSID should be configurable by the node operator.

Reply:
Yeah I did it to fit on the OLED better, but yeah that's not a great reason.